dev/doc/README.md

```diff
@@ -6,6 +6,40 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
+## Configuring Dune caching and default verbosity
+
+There are several configuration settings that you can control globally
+with a Dune configuration file (see `man dune-config` to learn
+more). This file is generally located in `~/.config/dune/config` and
+should start with the version of the Dune language used, e.g.:
+
+```
+(lang dune 2.0)
+```
+
+- You will get faster rebuilds if you enable Dune caching. This is
+  true in all cases, but even more so when using the targets in
+  `Makefile.dune` (see below).
+
+  To set up Dune caching, you should append the following line to your
+  Dune configuration file:
+
+  ```
+  (cache enabled)
+  ```
+
+  Note that by default, Dune caching will use up to 10GB of disk size.
+  See the [official documentation](https://dune.readthedocs.io/en/stable/caching.html#on-disk-size)
+  to learn how to change the default.
```

Comment:
10 GB seems rather piggy considering that the build directory is only about 1GB.  I wonder how well dune handles building in multiple source trees (serially or in parallel).

Changing the default is so simple IIUC you might as well mention it here, i.e. `(cache-trim-size 999MB)`.

The webpage link here and the man page mentioned on line 13 could go at the very beginning of the section, I think.

Reply:
For some people the cache is overkill, others have cache sizes in the TiB , so indeed YMMV here. 10 GiB is a good default for the regular OCaml developer.

> I wonder how well dune handles building in multiple source trees (serially or in parallel).

Not sure what you mean by this, I'll be happy to answer over Zulip tho.

Reply:
Note that the cache had some known issues on windows, I understand it has much improved lately, but TTBOMK it is still experimental on that platform.

Reply:
FWIW, I use WSL2 (Ubuntu VM on Windows), not native Windows.